lib/httpErrors.d.ts

```diff
@@ -1,12 +1,6 @@
-export interface HttpError extends Error {
-  status: number;
-  statusCode: number;
-  expose: boolean;
-  headers?: {
-    [key: string]: string;
-  };
-  [key: string]: any;
-}
+import { HttpError } from "http-errors";
```

Comment:
@climba03003 This is not just an interface. Its an actual class

Reply:
Then I dont see why we cant do

```
interface HttpErrorConstructor {
        (msg?: string): HttpError;
        new(msg?: string): HttpError;
}
```

Then export it.

Reply:
Is this better than using upstream types ? I mean I get that you want reduce deps but is duplicating upstream types the preferred approach for you ?

Reply:
@climba03003 I have updated the PR accordingly. Please do take a look